### PR TITLE
Format the double key display better

### PR DESCRIPTION
**Pull request by wushin**

### CODE REVIEW

**Comment by mekolat** [comment]
***"Just use the [up] to move up"*** sounds weird.. you should either remove "the" or add "button":
* Just use [up] to move up
* Just use the [up] button to move up

**Comment by wushin** [comment]
@mekolat ding fries are done.

File: world/map/npc/029-2/sorfina.txt [hunk 1]
```diff
@@ -57,8 +57,8 @@ L_Tut:
 L_Carpet:
     mes "[Sorfina]";
     mes "\"Try to take a few steps and walk to that carpet over there.";
-    mes "Just use the ###keyMoveUp; to move up, press ###keyMoveDown; to move down, ";
-    mes "press ###keyMoveLeft; to move left, press ###keyMoveRight; to move right ";
+    mes "Just use the [###keyMoveUp;] to move up, press [###keyMoveDown;] to move down, ";
```
Comment (mekolat) [inline world/map/npc/029-2/sorfina.txt]: @wushin ??

Comment (wushin) [inline world/map/npc/029-2/sorfina.txt]: What? dont just ??

Comment (mekolat) [inline world/map/npc/029-2/sorfina.txt]: it still says "Just use the [up] to move up"

**Comment by wushin** [comment]
Doh, my bad I missed git adding the file. I was wondering what was up with github

**Pull request merged**